### DISCUSSION TITLE
details on transactional commit mode in section 13.1

### DISCUSSION THREAD

**Discussion by pdeole** [confluentinc/parallel-consumer#324] [Q&A] [answered]
Hi @astubbs  and team

can someone elaborate more on section 13.1 and how the transactional offset commit mode works? 
Perhaps an example would help on the same

**Comment by astubbs** [comment]
Yes, that area does need some love. Can I ask, what your domain or intended use case is? That might help me explain it better.

**Comment by astubbs** [comment, marked as answer]
Ok @pdeole , I've taken a stab at documenting the feature better, can you take a look at PR: #355 and tell me if that clarifies things for you?
- #355

You can see the rendered new readme section here: 
https://github.com/astubbs/parallel-consumer/tree/improvements/transaction-docs#apache-kafka-eos-transaction-model

**Comment by pdeole** [comment]
Sure @antony, will take a look

**Reply by astubbs** [reply]
Ok that turned out to be a beast :) But it's solid now - #355 is merged.
